### ISSUE TITLE
508-defect-4 [SEMANTIC MARKUP, COGNITION]: Consider visually and semantically splitting longer labels into a label and descriptive hint text using aria-describedby

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25174]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

Some labels are particularly verbose including:
- Step 1 of 4: Veteran Detail's "Do you have a representative-- like an accredited attorney, a claims agent..."
- Step 3 of 4: Board review options

## Acceptance Criteria
- [ ] Descriptive content in long labels are split apart from the label
- [ ] Any descriptive text split away from the label are linked to the input using `aria-describedby`

## WCAG or Vendor Guidance (optional)

> A label should be concise, where a description is intended to provide more verbose information.
https://www.w3.org/TR/wai-aria/#aria-describedby

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/14154792/119403455-60d68380-bcac-11eb-9194-4ddea64a268f.png)
![image](https://user-images.githubusercontent.com/14154792/119404878-7187f900-bcae-11eb-9715-9ccb26580e2c.png)
